src/borg/testsuite/repository.py

```diff
@@ -94,8 +91,10 @@ def repo_dump(repository, label=None):
     print()
 
 
-def test_basic_operations(repository):
-    with repository:
+@pytest.mark.parametrize("fixture", ["repository", "remote_repository"])
+def test_basic_operations(fixture, request):
+    repo = request.getfixturevalue(fixture)
+    with repo as repository:
```

Comment:
Interesting. Not sure if that is the best way to accomplish that, maybe @RonnyPfannschmidt knows better.

Reply:
the lazy_fixtures plugin has a way to refer to other fixtures as parameters safely,
i generally recommend against using getfixturevalue and request inside of a test function

Reply:
doesn't look maintained, so we'll do it without it for now.